botocore/handlers.py

```diff
@@ -441,8 +447,33 @@ def switch_host_with_param(request, param_name):
      base64_encode_user_data),
     ('before-parameter-build.route53', fix_route53_ids),
     ('before-parameter-build.glacier', inject_account_id),
+
+    # Glacier documentation customizations
     ('docs.*.glacier.*.complete-section',
-     AutoPopulatedParam('accountId').document_auto_populated_param),
+     AutoPopulatedParam('accountId', 'Note: this parameter is set to "-" by \
+                         default if no value is not specified.')
+     .document_auto_populated_param),
     ('docs.*.glacier.*.complete-section',
-     AutoPopulatedParam('checksum').document_auto_populated_param)
+     AutoPopulatedParam('checksum').document_auto_populated_param),
+    # UserData base64 encoding documentation customizations
+    ('docs.*.ec2.RunInstances.complete-section', document_base64_encoding()),
+    ('docs.*.autoscaling.CreateLaunchConfiguration.complete-section',
+     document_base64_encoding()),
+    # EC2 CopySnapshot documentation customizations
+    ('docs.*.ec2.CopySnapshot.complete-section',
+     AutoPopulatedParam('PresignedUrl').document_auto_populated_param),
+    ('docs.*.ec2.CopySnapshot.complete-section',
+     AutoPopulatedParam('DestinationRegion').document_auto_populated_param),
+    # S3 SSE documentation modifications
+    ('docs.*.s3.*.complete-section',
+     AutoPopulatedParam('SSECustomerKeyMD5').document_auto_populated_param),
+    # The following S3 operations cannot actually accept a ContentMD5
+    ('docs.*.s3.*.complete-section',
+     HideParamFromOperations(
+         's3', 'ContentMD5',
```

Comment:
I noticed that there is a similiar customization for SSECustomerKey and SSECustomerKeyMD5. Can we add documentation for that as well? I know the first time I tried using sse it took me a fair amount of time to realize a customization is being applied to the parameters.